resources/templates/account_password.html

```diff
@@ -25,6 +25,15 @@ <h2>Create your Wallet</h2>
 </div>
 {% end %}
 
+{% block stepper %}
+<div class="coins">
+    <img class="transparent" src="{{ static_url('images/eth.png') }}" alt="Raiden Network" />
+    <img class="transparent" src="{{ static_url('images/rdn.png') }}" alt="Raiden Network" />
+    <img class="transparent" src="{{ static_url('images/dai.png') }}" alt="Raiden Network" />
+</div>
+Step 1 of 6: Create your Wallet...
```

Comment:
Can you write `Create your Raiden Account`?

Reply:
Ah forgot about that. Thanks!